.env

```
@@ -14,8 +14,9 @@ VITE_APP_ALKEMIO_DOMAIN=http://localhost:3000
 VITE_APP_LOG_ERRORS=true
 
+# Unified collaboration service (memo + whiteboard) — the browser opens
+# ws://<host>/collab/<documentId>?type=memo|whiteboard through the gateway
+# (Traefik/Oathkeeper → collaboration-service). Replaces the legacy Socket.IO
+# (whiteboard) + Hocuspocus (memo) endpoints.
 VITE_APP_COLLAB_URL=http://localhost:3000
-VITE_APP_COLLAB_PATH=/api/private/ws/socket.io
-
-VITE_APP_COLLAB_DOC_URL=http://localhost:3000
-VITE_APP_COLLAB_DOC_PATH=/api/private/hocuspocus
+VITE_APP_COLLAB_PATH=/collab
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The VITE_APP_COLLAB_PATH key should go before the VITE_APP_COLLAB_URL key

(UnorderedKey)